iib/workers/tasks/build.py

```diff
@@ -180,10 +182,12 @@ def _finish_request_post_build(
     payload = {
         'arches': list(arches),
         'index_image': index_image,
-        'state': 'complete',
-        'state_reason': 'The request completed successfully',
+        'state': 'in_progress',
+        'state_reason': 'Updated the index image',
```

Comment:
Can we please make this `state_reason` a bit more verbose? How about creating a variable called `state_reason` in the beginning and updating it depending upon which branch gets executed?

@mprahl Thoughts?

Reply:
I like that idea.

Reply:
@yashvardhannanavati Could you please elaborate on what you mean by "which branch gets executed"?

Reply:
So if the first `if` branch gets executed, maybe you can add the message in `log.info` to the state_reason. Similar if the `elif` block gets executed. Does that make sense?

Reply:
Got that, earlier I thought you meant changing the state_reason based on what part of add request was executing and I got confused by that. Thanks for clarifying

Reply:
Got that, earlier I thought you meant changing the state_reason based on what part of add request was executing and I got confused by that. Thanks for clarifying